ocrd_utils/ocrd_utils/constants.py

```diff
@@ -38,6 +38,7 @@
 
 MIME_TO_EXT = {
     'image/tiff': '.tif',
+    'image/tif': '.tif',
```

Comment:
That's not a valid MIME type, right? You only want to be a bit tolerant, right?

Reply:
Yes, b/c it is used (wrongly) like that sometimes.